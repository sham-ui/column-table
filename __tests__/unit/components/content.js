import Content  from '../../../src/components/content.sfc';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( Content, {} );
    expect( meta.toJSON() ).toMatchSnapshot();
} );

it( 'getter', () => {
    const meta = renderer( Content, {
        item: 'Foo',
        index: 1,
        getter: ( item, index ) => item + index
    } );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
