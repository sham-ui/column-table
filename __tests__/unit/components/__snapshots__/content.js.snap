// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getter 1`] = `
Object {
  "Options": Object {
    "getter": [Function],
    "index": 1,
    "item": "Foo",
  },
  "html": "Foo1",
}
`;

exports[`renders correctly 1`] = `
Object {
  "Options": Object {
    "getter": [Function],
  },
  "html": "",
}
`;
